gapic/src/main/com/google/gapid/perfetto/views/CounterPanel.java

```diff
@@ -95,7 +95,10 @@ protected void renderTrack(RenderContext ctx, Repainter repainter, double w, dou
       }
 
       CounterInfo counter = track.getCounter();
-      double min = counter.range.min, range = counter.range.range();
+
+      //TODO: Move all the calculations below in the backend
+      double min = counter.min;
+      double range = counter.range.range() - min;
```

Comment:
Does this imply range is actually "max - 0" instead of "max - min"?

Reply:
`Range.range()` does return `max - min`. `range.min`, which was used before instead of `counter.min` was `max(counter.min, 0)` so that the bottom of the y-axis is never > zero. This was done on purpose, so I think this change is not the right change, at least not for all counters.